### PR TITLE
Default config & template

### PR DESCRIPTION
**Pull request by robertkowalski**
- Add default template
 - Add default config

Makes it easy to get started:

After cloning the repo, install the dependencies and create a configuration:

```
npm install
cp couch_email_authrc-dist couch_email_authrc
```

Change the configuration saved in `couch_email_authrc` to your needs

### CODE REVIEW

File: .couch_email_authrc-dist [hunk 1]
```diff
@@ -0,0 +1,16 @@
+host = 127.0.0.1
+port = 3000
+couchDbBaseUrl = http://admin:secret@localhost:5984
+usersDb = _users
+sessionTimeout = 1
```
Comment (KlausTrainer) [inline .couch_email_authrc-dist]: This is the token expiration time in seconds.  We should either leave it out, or use a sensible default.

Btw., the name `sessionTimeout` is misleading.  We should rename this configuration to something more self-explanatory like e.g. `tokenExpiryTime`.  I'll address that in a separate issue.

File: .couch_email_authrc-dist [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+redirectLocation = http://localhost:5984/
+
+[smtp]
+  port = 2525
```
Comment (KlausTrainer) [inline .couch_email_authrc-dist]: Port `2525` is no sensible default.  Port `25` should be fine in most cases.

File: .couch_email_authrc-dist (hunk 1 above)
Comment (KlausTrainer) [inline .couch_email_authrc-dist]: For configuration settings where the meaning of the value is not obvious from the context, we should provide a short explanation in a comment.

For instance:

```ini
tokenExpiryTime = 1 ; the time in seconds the authentication token (which is part of the sign-in link) is valid 
```

**Comment by robertkowalski** [comment]
great feedback, fixed!

File: .couch_email_authrc-dist [hunk 3]
```diff
@@ -0,0 +1,22 @@
+host = 127.0.0.1
+port = 3000
+couchDbBaseUrl = http://admin:secret@localhost:5984
+usersDb = _users
+
+; the time in seconds the authentication token
+; (which is part of the sign-in link) is valid
+tokenExpiryTime = 1
```
Comment (KlausTrainer) [inline .couch_email_authrc-dist]: Your description in the comment is great, but the value is still a bit too low ;)
Something like `3600` (i.e., one hour) would be reasonable, I guess.

**Comment by robertkowalski** [comment]
lol my apologies klaus trainer, next try.

**Comment by KlausTrainer** [comment]
No worries! Thanks :)

**Pull request merged**